sale_pricelist_auto_update/models/res_partner.py

```diff
@@ -38,7 +43,7 @@ def _update_current_pricelist(self):
         amount = hist_recs.sorted(
             key=lambda r: r.amt_total)[-1].amt_total
         group = self.property_product_pricelist.pricelist_group_id
-        if group:
+        if group and not self.fix_pricelist:
```

Comment:
Check the attribute before update the pricelist while history is also updated (created)